editor/src/clj/editor/gui.clj

```diff
@@ -1665,7 +1665,9 @@
             (set (layout-property-setter inner-radius)))
   (property perimeter-vertices g/Int (default (protobuf/default Gui$NodeDesc :perimeter-vertices))
             (dynamic error (g/fnk [_node-id perimeter-vertices] (validate-perimeter-vertices _node-id perimeter-vertices)))
-            (dynamic edit-type (layout-property-edit-type perimeter-vertices {:type g/Int}))
+            (dynamic edit-type (layout-property-edit-type perimeter-vertices {:type g/Int
+                                                                              :min perimeter-vertices-min
+                                                                              :max perimeter-vertices-max}))
```

Comment:
I think we always use a slider when we have a min & max, except in this case. I didn't change that, because I wanted to support value clamping on mouse drag anyway. If the gap is huge, or if we only have a min or max, it doesn't make sense to use a slider.